README.md

```diff
@@ -14,6 +14,26 @@ it, simply add the following line to your `Podfile`:
 pod "Segment-Appboy"
 ```
 
+## Usage
+
+After adding the dependency, you must register the integration with our SDK. To do this, import the Appboy integration in your AppDelegate:
+
+
+```
+#import <Segment-Appboy/AppboyIntegrationFactory.h>
+```
+
+And add the following lines:
+
+```
+NSString *const SEGMENT_WRITE_KEY = @" ... ";
+SEGAnalyticsConfiguration *config = [SEGAnalyticsConfiguration configurationWithWriteKey:SEGMENT_WRITE_KEY];
+
+[config use:[SEGAppboyIntegrationFactory instance]];
+
+[SEGAnalytics setupWithConfiguration:config];
+```
```

Comment:
Can you just add this line with our documentation link at the end?
`Please see our documentation for more info: https://segment.com/docs/integrations/appboy/#ios`